deploy/csv-templates/ics-operator.csv.yaml.in

```diff
@@ -382,6 +432,8 @@ spec:
           - patch
           - update
           - watch
+          - deletecollection
```

Comment:
No need to add the permissions where wildcards were not being used before. In this case, there was no wilcard for the verb.